vuu-ui/packages/vuu-shell/src/layout-management/SaveLayoutPanel.tsx

```diff
@@ -71,14 +71,6 @@ export const SaveLayoutPanel = (props: SaveLayoutPanelProps) => {
           <FormField className={formField}>
             <FormFieldLabel>Group</FormFieldLabel>
             <ComboBox
-              ListProps={{
-                style: {
-                  zIndex: 10000,
-                  border: "1px solid #777C94",
-                  borderRadius: 10,
-                  boxSizing: "border-box"
-                }
-              }}
```

Comment:
Are these properties defined somewhere else now? I seem to remember the `zIndex` being needed to make the dropdown list visible.

Reply:
I think Steve has added z-index to the portal style now so it is no longer needed here

Reply:
steve's commit: https://github.com/finos/vuu/pull/851/commits/ea9adfa126c7ca4f6068ce99aef06049ca1139e0#diff-b05c48851b2d6a1842dc5ac85ea176740d54bf628287cc51ad250dcb32cd8bec